README

```diff
@@ -3,3 +3,23 @@ Simple support for Go testing with a live Postgres database
 
 This package makes it straightforward to write tests that connect
 to a fresh Postgres database created for the tests.
+
+## Set up ##
+In order to use pgtest, you'll need to setup your postgres user as follow:
```

Comment:
s/pgtest/postgrestest/

Also, I think I'd probably move this comment to the package doc comment so it's available in godoc. (optional for bonus points: mirror package docs in README.md).

How about something like this?

    In order to use postgrestest, you'll need to set up connection string parameters
    that allow the client to create schemas and tables (see New for more details).

    Here is a way to set up postgres to contain a new user named "someuser" with
    password "somepassword" that has the required access:
    
        sudo -u postgres createuser someuser
        sudo -u postgres psql -c "alter user someuser with password 'somepassword'"
        sudo -u postgres psql -c "create database someuser"
        sudo -u postgres psql -c "grant all on database someuser to someuser"
        export PGUSER=someuser
        export PGPASSWORD=somepassword

Reply:
Or:
````
sudo -u postgres createuser <someuser> -P
Enter password for new role: <somepassword>
sudo -u postgres createdb someuser -O someuser
export PGUSER=someuser
export PGPASSWORD=somepassword
```